src/test/java/org/cactoos/ScalarTest.java

```diff
@@ -23,27 +23,28 @@
  */
 package org.cactoos;
 
+import org.cactoos.scalar.NoNulls;
 import org.junit.Test;
 
 /**
- * Test case for {@link Scalar.NoNulls}.
+ * Test case for {@link NoNulls}.
  * @since 0.11
  * @checkstyle JavadocMethodCheck (500 lines)
  */
 public final class ScalarTest {
 
     @Test(expected = IllegalArgumentException.class)
```

Comment:
@dgroup Let's match the class and message exception. You can use something like:
```
@Rule
public final ExpectedException exception = ExpectedException.none();
```
to achieve it (and do that for all exceptions).

Reply:
@fabriciofx The usage of ExpectedException is out of the scope of this task.
This task is related to the extraction of NoNulls interfaces from the `cactoos` primitives.
I don't want to fix this because:
1. The modification of tests to fix the issues which are outside of this task is prohibited;
2. The way you are suggesting isn't approved by @llorllale and will be fixed in future. I don't see any profit from this action now, honestly.